presets/wikimedia.json

```diff
@@ -65,5 +65,6 @@
     "disallowNewlineBeforeBlockStatements": true,
     "validateLineBreaks": "LF",
     "validateQuoteMarks": "'",
-    "validateIndentation": "\t"
+    "validateIndentation": "\t",
+    "requireSpacesInForStatement": true
```

Comment:
This seems already covered by `requireSpacesInsideParentheses`. Without that rule, we'd need a rule for `if`, `while`, `switch`, etc. as well. Not just `for`.